epitome/functions.py

```diff
@@ -46,21 +46,23 @@
 # to load in positions file
 import multiprocessing
 
+################## LOADING DATA ######################
+
 def download_and_unzip(url, dst):
     '''
     Downloads a url to local destination, unzips it and deletes zip.
 
-    :param str url: url to download.
-    :param str dst: local absolute path to download data to.
+    :param url: url to download.
```

Comment:
why are these removed?

Reply:
Hm not sure if it got lost in the git rebase-- added back